openrouteservice/src/main/java/heigit/ors/routing/RoutingRequest.java

```diff
@@ -1,202 +1,214 @@
-
-	public void setIncludeInstructions(boolean includeInstructions) {
-		_includeInstructions = includeInstructions;
-	}
-
-	public DistanceUnit getUnits() {
-		return _units;
-	}
-
-	public void setUnits(DistanceUnit units) {
-		_units = units;
-	}
-
-	public String getGeometryFormat() {
-		return _geometryFormat;
-	}
-
-	public void setGeometryFormat(String geometryFormat) {
-		_geometryFormat = geometryFormat;
-	}
-
-	public boolean getGeometrySimplify() { return _geometrySimplify; }
-
-	public void setGeometrySimplify(boolean geometrySimplify) { _geometrySimplify = geometrySimplify; }
-
-	public String getLanguage() {
-		return _language;
-	}
-
-	public void setLanguage(String language) {
-		_language = language;
-	}
-
-	public RouteInstructionsFormat getInstructionsFormat() {
-		return _instructionsFormat;
-	}
-
-	public void setInstructionsFormat(RouteInstructionsFormat format) {
-		_instructionsFormat = format;
-	}
-
-	public int getExtraInfo() {
-		return _extraInfo;
-	}
-
-	public void setExtraInfo(int extraInfo) {
-		_extraInfo = extraInfo;
-	}
-
-	public Boolean getIncludeElevation() {
-		return _includeElevation;
-	}
-
-	public void setIncludeElevation(Boolean includeElevation) {
-		this._includeElevation = includeElevation;
-	}
-
-	public Boolean getIncludeGeometry() {
-		return _includeGeometry;
-	}
-
-	public void setIncludeGeometry(Boolean includeGeometry) {
-		this._includeGeometry = includeGeometry;
-	}
-
-	public String[] getAttributes() {
-		return _attributes;
-	}
-
-	public void setAttributes(String[] attributes) {
-		_attributes = attributes;
-	}
-	
-	public boolean hasAttribute(String attr) {
-		if (_attributes == null || attr == null)
-			return false;
-
-		for (int i = 0; i< _attributes.length; i++)
-			if (attr.equalsIgnoreCase(_attributes[i]))
-				return true;
-
-		return false;
-	}
-
-	public boolean getConsiderTraffic(){
-		return this._searchParameters.getConsiderTraffic();
-		
-	}
-
-	public int getLocationIndex() {
-		return _locationIndex;
-	}
-
-	public void setLocationIndex(int locationIndex) {
-		_locationIndex = locationIndex;
-	}
-
-	public Boolean getIncludeManeuvers() {
-		return _includeManeuvers;
-	}
-
-	public void setIncludeManeuvers(Boolean includeManeuvers) {
-		_includeManeuvers = includeManeuvers;
-	}
-
-	public boolean getContinueStraight() {
-		return _continueStraight;
-	}
-
-	public void setContinueStraight(boolean continueStraight) {
-		_continueStraight = continueStraight;
-	}
-
-	public boolean getIncludeRoundaboutExits() {
-		return _includeRoundaboutExits;
-	}
-
-	public void setIncludeRoundaboutExits(boolean includeRoundaboutExits) {
-		_includeRoundaboutExits = includeRoundaboutExits;
-	}
-
-	public boolean getSuppressWarnings() {
-		return _suppressWarnings;
-	}
-
-	public void setSuppressWarnings(boolean suppressWarnings) {
-		_suppressWarnings = suppressWarnings;
-	}
-
-	public boolean isValid() {
-		return !(_coordinates == null);
-	}
-}
+/*  This file is part of Openrouteservice.
```

Comment:
why is the whole of this file being marked as changed?

Reply:
Actually I have no idea. Maybe it is a GitHub thing. In my local IDE it is all fine...

Reply:
We should make sure that we don't do these sort of things in a commit for a specific feature as it makes it lamost impossible to see the functionality that has been introduced or changed - I have to go through the file line by line to look for differences

Reply:
Sure, I get that. I can overwrite the changes and add that to a unique commit?!

Reply:
It's OK for this one as I can see where things changed as it is a smaller file, but if it was a bigger file then it wouldn't work